apps/flipper_web/pubspec.yaml

```
@@ -27,6 +27,7 @@ dependencies:
   firebase_auth: ^6.1.0
   firebase_core: ^4.1.1
   ditto_live: ^4.13.1
+  ditto_live: ^4.13.1
```

YAMLlint (1.37.1)

[error] duplication of key "ditto_live" in mapping

(key-duplicates)